pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Instead of adding `"Programming Language :: Python :: 3.11"` would you mind removing everything except `"Programming Language :: Python :: 3 :: Only"` here?

It feels annoying keep this up-to-date and duplicate information where the real source of truth is the `requires-python` field.